sycl/test/basic_tests/image_accessor_readwrite.cpp

```diff
@@ -1,8 +1,10 @@
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: env SYCL_DEVICE_TYPE=HOST %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
-// RUNx: %GPU_RUN_PLACEHOLDER %t.out
-// RUNx: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// TODO: No CUDA image support
+// XFAIL: cuda
```

Comment:
```suggestion
// UNSUPPORTED: cuda
```

I think it is better not to launch test at all if it is not expected to work right now

Reply:
Done.